### PR TITLE
[Feedback request] 🔧 container is-fluid

### PR DESCRIPTION
**Pull request by roiLeo**
This is for testing purposes only 🧪 
ping @exezbcz 

## Context

- [x] ref #4857
- [x] container size for Navbar
- [x] container size for /explore pages

## Preview

https://deploy-preview-5089--koda-nuxt.netlify.app/bsx/explore/collectibles

### CODE REVIEW

**Comment by yangwao** [comment]
actually, when netlify first pick ups PR and you move it to draft, when new commit event is emitted, it builds it anyway even in draft...

**Comment by vikiival** [comment]
Please make this as draft

**Comment by vikiival** [comment]
👀 can I close?

**Comment by roiLeo** [comment]
> 👀 can I close?

I'm waiting for @exezbcz feedback to push changes

**Comment by exezbcz** [comment]
@roiLeo hmm, it's better but still its something missing 
- I am also not completely sure what it does, the margins are now flexible or? 
- I can see that on bigger screens the content is stretching now - There is no dynamic grid yet so the collection cards are for example stretched. 
- but for example on 1240px the margins are looking worse than on current beta.
- there seems to be another 32px padding added
image

the objective was to lower the margins on certain screens, or best, make it unified for desktop and mobile, mobile has fixed 20px afaik. 

do I understand this correctly? Correct me please if I am wrong :D

**Comment by roiLeo** [comment]
> I am also not completely sure what it does, the margins are now flexible or?
> I can see that on bigger screens the content is stretching now 

we can set fixed margin (left & right) for container, on this preview it's set as 32px and content will fulfill container

> There is no dynamic grid yet so the collection cards are for example stretched.

not related but we can rework resizing of collections cards

> but for example on 1240px the margins are looking worse than on current beta.

Looks oki?
Capture d’écran 2023-03-06 à 4 49 33 PM

> there seems to be another 32px padding added

This will be default padding (can be changed) between the content and the edge of the window

**Comment by exezbcz** [comment]
@roiLeo yup, that is better now. 
I think something like 40px margin for desktop would be good, and then 20px for specific lower breakpoints - 1024?

**Comment by vikiival** [comment]
cc @exezbcz need your visual 👁️ 👁️ 👁️

File: styles/global.scss [hunk 1]
```diff
@@ -101,6 +101,11 @@ body {
   }
 }
 
+.container.is-fluid {
+  padding-left: 40px;
```
Comment (vikiival) [inline styles/global.scss:105]: maybe use `rem` instead of `px`?

File: styles/global.scss [hunk 2]
```diff
@@ -113,6 +118,10 @@ body {
     padding-right: 10px !important;
     margin: 0 -12px !important;
   }
+  .container.is-fluid {
```
Comment (vikiival) [inline styles/global.scss:121]: likewise

**Review by vikiival — CHANGES_REQUESTED** [review]
Smol

**Comment by exezbcz** [comment]
yup, its okay, but its not working on landing page - is that expected? 

and with bigger items on explore - but that would be fixed individually no? 
image

**Comment by roiLeo** [comment]
> yup, its okay, but its not working on landing page - is that expected?

Now should be testable on landing page

> and with bigger items on explore - but that would be fixed individually no? 

yeah we could change fixed width to make it more responsive

**Comment by exezbcz** [comment]
@roiLeo it definitely makes sense for collection page and for explore page 
- We should apply it to the rest so its consistent, wdyt?

**Comment by roiLeo** [comment]
> We should apply it to the rest so its consistent, wdyt?

Yes! Note that we will keep some pages at half size layout (teleport, create..) for now

**Comment by exezbcz** [comment]
@roiLeo yup, It makes no sense making teleport full width, maybe once it is portfolio page? 👀 

Thank you for the changes!

**Comment by exezbcz** [comment]
looking good! We will need to change the carousel on landing but otherwise solid.

**Comment by yangwao** [comment]
mergeable? would be nice to push it out for release

**Review by vikiival — APPROVED** [review]

**Comment by vikiival** [comment]
pay 50 usd

**Comment by roiLeo** [comment]
I didnt had Time to test it on a large screen :/. How does it look ?

**Pull request merged**